docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,167 @@
+
+Add these lines back to your `Cargo.toml`.
+```toml
+
+[dev-dependencies.primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'substrate-primitives'
+branch = 'v1.0'
+
+[dev-dependencies.runtime-primitives]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-primitives'
+branch = 'v1.0'
+
+[dev-dependencies.runtime-io]
+default_features = false
+git = 'https://github.com/paritytech/substrate.git'
+package = 'sr-io'
+branch = 'v1.0'
+```
+
+It will take a little experimentation to gain familiarity with what dependencies you need for each module you dream up. So the take-away from this section is not that `Cargo.toml` should always look like it does in the template. Rather, `Cargo.toml` needs to have the correct dependencies and you now know how to specify them.
+
+## Add your Runtime Module to your Node
+With our runtime module now compiling and passing tests (well, passing test anyway) we're ready to add it to our node template.
+
+First we need to add our newly-created crate as a dependency in the node runtime's `Cargo.toml`.
+
+```toml
+[dependencies.test_module]
+default_features = false
+path = "../../test_module"
+# Optionally specify a version
+# version = "0.1.0"
+```
+
+And just as before, we need to tell the module to only build its `std` feature when the runtime itself does.
+```toml
+std = [
+  ...
+  'test_module/std',
+]
+```
+
+Next we'll update `runtime/lib.rs` in modular-chain to actually use the new runtime module. The lines that bring in the template module that is coded directly in the node runtime are still there, so we just need to update them in three places.
+
+1. Remove `mod template` as we've already brought that dependency in via `Cargo.toml`
+2. Update the trait implementation
+```rust
+/// Used for the module test_module
+impl test_module::Trait for Runtime {
+	type Event = Event;
+}
+```
+
+3. Update `construct_runtime`
+```rust
+TestModule: test_module::{Module, Call, Storage, Event<T>},
+```
+
+## Appreciate your Work
+At this point you have the template module packaged up as it's own crate, and included in your `modular-chain` runtime.
+
+Build the `modular-chain` node with
+```bash
+./scripts/build.sh
+cargo build --release
+```
+
+And run your node with
+```bash
+# Clear the data directory in case you've done this before
+./target/release/modular-chain purge-chain --dev
+
+# Start the chain
+./target/release/modular-chain --dev
+```
+
+Finally, start the [Apps UI](https://polkadot.js.org/apps/#/explorer) to confirm that the module is working as expected. If you aren't familiar with the Apps UI, you will need to navigate to the settings tab, and select 'Local Node'.
```

Comment:
I think the lack of a "better link" in this section means we probably need a short guide on interacting with the Polkadot UI. We should talk to Jaco and team.

Reply:
Agreed. I was thinking of putting something like that in the "ecosystem tools" section of devhub. I would prefer not to block this PR on that though.